.env.example

```diff
@@ -13,10 +13,11 @@ KEYCLOAK_REDIRECT_URI=http://localhost:4000/auth
 STRIPE_PUBLIC_API_KEY=ENTER_STRIPE_PUBLIC_KEY_HERE
 OAUTH2_AUTH_URL=http://localhost:4444/oauth2/auth
 OAUTH2_TOKEN_URL=http://hydra:4444/oauth2/token
-HYDRA_ADMIN_URL=http://hydra:4445
```

Comment:
Unrelated to this PR, but discovered this unused env var. It shouldn't be here anymore.